README.md

```diff
@@ -6,48 +6,45 @@
 ![](https://img.shields.io/github/issues-pr/bluewave-labs/bluewave-uptime)
 ![](https://img.shields.io/github/issues/bluewave-labs/bluewave-uptime)
 
```

Comment:
Next time please make a separate pull request for updating the readme.

A pull request should be very focused and only deal with one feature / fix / update at a time.

Imagine we need to undo this README change for whatever reason.  If it was a separate PR we could just revert that PR.

Now since this PR has more than one feature / fix our job is more difficult.

Reply:
@ajhollid 
This is not my modification!
I just fetched, and this among changed files!
I have only added and modified only and only 3 files:

`Client/src/Components/CreateMaintenanceWindow/index.jsx`
`Client/src/Components/CreateMaintenanceWindow/index.css`
`App.js`

These are the only files I've worked on. Don't know why other files are added. I've just made a sync on the branch, and I guess other files are added through that!

Reply:
The issue is probably that you've merged `master` in to your feature branch:

![image](https://github.com/user-attachments/assets/d494b4c3-0ea1-48be-aa07-83207aacf3d7)

This is not something that we should ever do, we should only ever merge `develop` into `feature`.

If you merge `master` into `feature` and then `feature` into `develop` then you have effectively merged `master` in to `develop`, which is something we never want to do.

The fix here is simple, just take your chagnes and move them to a new branch that you branch off of `develop`.

1.  `git checkout develop`
2.  `git checkout -b <my_feature>`
3.  Make your changes, commit, and push

This will ensure a clean git commit history and will make sure you don't have unwanted changes in your code :+1: